modules/infra/cli/nexthop.c

```diff
@@ -59,7 +59,7 @@ static cmd_status_t show_config(const struct gr_api_client *c, const struct ec_p
 }
 
 static cmd_status_t nh_add(const struct gr_api_client *c, const struct ec_pnode *p) {
-	struct gr_nh_add_req req = {.exist_ok = true};
+	struct gr_nh_add_req req = {.exist_ok = true, .nh.origin = GR_NH_ORIGIN_USER};
```

Comment:
When creating routes, we don't specify any origin. Could you do the same for nexthops maybe?

Reply:
The API is there but grcli isn't using it.